bindata/oauth-apiserver/deploy.yaml

```diff
@@ -39,6 +39,10 @@ spec:
           command: ['sh', '-c', 'chmod 0700 /var/log/oauth-apiserver && touch /var/log/oauth-apiserver/audit.log && chmod 0600 /var/log/oauth-apiserver/*']
           securityContext:
             privileged: true
+          resources:
+            requests:
+              cpu: 15m
+              memory: 50Mi
```

Comment:
what do we have in openshift-apiserver?

Have you checked a real cluster? Console should show values.

Reply:
This specific container just comes and goes so it's hard to retrieve what it really attempts to request. I wasn't able to get that with `oc adm top pod --containers`, nor with the console.

The openshift-apiserver does not seem to be setting the requests here either.

Reply:
oh, didn't see it's not the main container. All fine.